scripts/mbedtls_dev/bignum_core.py

```diff
@@ -16,20 +16,19 @@
 
 import random
 
-from abc import ABCMeta
 from typing import Dict, Iterator, List, Tuple
 
 from . import test_case
 from . import test_data_generation
 from . import bignum_common
 
-class BignumCoreTarget(test_data_generation.BaseTarget, metaclass=ABCMeta):
-    #pylint: disable=abstract-method
+class BignumCoreTarget(test_data_generation.BaseTarget):
```

Comment:
Having looked superficially at the changes here but not dived into the details, this looks sensible to me.

To the Python experts: what to we lose by not giving certain classes a metaclass anymore?

Reply:
In short @yanesca described it accurately. This is more about maintaining the metaclass properties, as such the strong enforcement that the abstraclasses are implemented by the children.

The rule of thumb is that:
* If you want this class to be inheritted and children have to implement the abstractclasses ==> use metaclass=ABCMeta
* If you want this class to be standlone or the implementation of the abstractmethods defined in it, to be suffient for it's children you can remove the `metaclass=ABCMeta`